src/examples/spaceshooter/effects/FXStarfieldQuarks.ts

```diff
@@ -6,23 +6,18 @@ import { Object3D } from "three";
 
 // Define the type for the methods you will expose via ref
 
-  export const StarFieldFX =((props, ref) => {
+  export const StarFieldFX =(() => {
     const [batchRenderer] = useState(new BatchedRenderer());
```

Comment:
should this be moved into the useEffect? ... looks a bit out of place having a state without ever needing to set it, and you probably don't want to be instantiating a new one on each render